app/components/collections/works_component.html.erb

```diff
@@ -1,4 +1,4 @@
-<table class="table" data-controller="datatables">
+<table class="table" id="worksTable" data-controller="datatables" data-datatables-selector-value="works">
```

Comment:
Note: adding this `data-tables-selector-value` is required so that more than one table can be on a page. Otherwise the first table with the given class is styled.